test/com/google/enterprise/adaptor/database/ResponseGeneratorTest.java

```diff
@@ -409,6 +599,36 @@ public void testBlobColumnModeContentTypeCol() throws Exception {
   }
 
   @Test
+  public void testBlobColumnModeContentTypeCol_nullValue() throws Exception {
+    String content = "hello world";
+    executeUpdate(
+        "create table data(id int, content blob, contentType varchar)");
+    String sql =
```

Comment:
Join lines, plus other opportunities below. Not sure if they were on purpose or not.